yarn.lock

```diff
@@ -514,7 +514,7 @@
     "@types/warning" "^3.0.0"
     warning "^4.0.3"
 
-"@iota/types@^1.0.0-beta.9":
+"@iota/types@^1.0.0-beta.8", "@iota/types@^1.0.0-beta.9":
```

Comment:
huh?

Reply:
Hmmm it seems related to [this](https://github.com/iotaledger/firefly/blob/stardust-develop/package.json#L9) :thinking: